openquake/calculators/base.py

```diff
@@ -982,7 +982,8 @@ def _read_risk3(self):
             # NB: get_station_data is extending the complete sitecol
             # which then is associated to the site parameters below
             self.station_data, self.observed_imts = \
-                readinput.get_station_data(oq, self.sitecol)
+                readinput.get_station_data(oq, self.sitecol,
+                                           duplicates_strategy='error')
```

Comment:
I would set `duplicates_strategy='avg'` so that the aristotle tests will get green.

Reply:
Done